index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
```

Comment:
@marcoscaceres or @kenchris  If everything else looks good beside the section nesting, would you want to fix the section nesting when merging?  THANK YOU for helping to land this.